src/components/KDropdown/KDropdown.vue

```diff
@@ -86,23 +80,27 @@ import KDropdownItem from './KDropdownItem.vue'
 import { ChevronDownIcon } from '@kong/icons'
 
 const props = defineProps({
-  appearance: {
-    type: String as PropType<Appearance>,
-    default: 'menu',
-    validator: (value: Appearance) => AppearanceArray.includes(value),
+  isSelectionMenu: {
+    type: Boolean,
+    default: false,
   },
-  buttonAppearance: {
+  appearance: {
     type: String as PropType<ButtonAppearance>,
     default: 'primary',
+    validator: (value: ButtonAppearance) => {
+      // @ts-ignore
+      if (value === 'menu' || value === 'selectionMenu') {
+        console.warn('KDropdown: some props have changed. Please see the migration guide for more details: https://alpha--kongponents.netlify.app/guide/migrating-to-version-9.html#kdropdownmenu')
+      }
+
+      return Object.values(ButtonAppearances).indexOf(value) !== -1
```

Comment:
suggestion: could this be simplified?
```suggestion
      return Object.values(ButtonAppearances).includes(value)
```